cmd/vpp-agent-ctl/main.go

```diff
@@ -388,25 +388,31 @@ func etcdDel(bDB *etcdv3.BytesConnectionEtcd, key string) {
 
 func createRoute(db keyval.ProtoBroker) {
 	routes := l3.StaticRoutes{}
-	routes.Ip = make([]*l3.StaticRoutes_Ip, 1)
-	routes.Ip[0] = new(l3.StaticRoutes_Ip)
-	routes.Ip[0].Description = "Description"
-	routes.Ip[0].VrfId = 0
-	routes.Ip[0].DestinationAddress = "10.1.1.2"
+	routes.Route = make([]*l3.StaticRoutes_Route, 1)
+	routes.Route[0] = new(l3.StaticRoutes_Route)
```

Comment:
could you format this code in golang style (not "C" like style). I mean embeding type instances {{{}}} in hierarchy.